rts/c/cuda.h

```diff
@@ -290,7 +337,9 @@ static const char *cuda_nvrtc_get_arch(CUdevice dev) {
     { 7, 0, "compute_70" },
     { 7, 2, "compute_72" },
     { 7, 5, "compute_75" },
-    { 8, 0, "compute_80" }
+    // NVRTC doesn't support compiling to 8.0, See nvrtc.h
+    { 8, 0, "compute_75" },
+    { 8, 6, "compute_75" }
```

Comment:
Wait, what?  Even if we want this, it should probably be a separate PR.  Have you tested that the CUDA backend works on these new GPUs?

Reply:
It's mostly a convenience thing for me, since I've a 8.6 compute capable device, so i do not have to manually edit programs after i compile them. As far as my testing go, I've not encountered any errors nor do i get any errors when i run a "stack test", but I agree it doesn't really belong to this PR.

Reply:
Then I'll make the change in master.

Reply:
I don't really understand the reference to NVRTC not supporting CC 8.0.  We've been using it for over a year.

Reply:
I think the most plausible explanation is that I'm using an outdated version of nvrtc